recipes/powershell/assets/README.md

```diff
@@ -1,23 +1,23 @@
-NetBackup Assets Service API Code Samples
+# NetBackup VMware Asset API Code Samples for PowerShell
 
-This directory contains PowerShell scripts demonstrating the use of NetBackup Asset Service APIs.
+This directory contains PowerShell scripts demonstrating the use of NetBackup Asset Service APIs for retrieving VMware assets.
 
-Disclaimer
+#### Disclaimer
 
 The samples are provided only for reference and not meant for production use.
 
-Executing the script
+#### Executing the recipes in PowerShell
 
-Pre-requisites:
+Prerequisites:
 
-NetBackup 8.3 or higher
-PowerShell 4.0 or higher
+-NetBackup 8.3 or higher
+-PowerShell 4.0 or higher

```

Comment:
This still isn't formatted as a list.

```suggestion
- NetBackup 8.3 or higher
- PowerShell 4.0 or higher
```

Reference: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
I already fixed this, and you put it back to what it was before. Please consider using a tool to preview your markdown text before pushing your commits.